gradle.properties

```diff
@@ -0,0 +1,12 @@
+org.gradle.jvmargs=-Xmx3G
+org.gradle.daemon=false
+
+minecraft_version=1.16.4
+mappings_version=20201028-1.16.3
+loader_version_range=[34,)
+forge_version=35.1.7
+mod_version=1.1.0a
+#This determines the minimum version of forge required to use
+# Only bump it whenever we need access to a feature in forge that is not available in earlier versions
+forge_version_range=[34.1.40,)
```

Comment:
One thing to note, the min version of forge can probably be lowered, maybe even to supporting more versions of 1.16 in general rather than just 1.16.3/4, I just didn't really do much testing on how large a range can be used, and just copy pasted the range from what Mekanism is currently using, as that is the other workspace I had open when starting this.